gecko_client/health_api_test.go

```diff
@@ -1,34 +1,56 @@
 package gecko_client
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
 )
 
 func TestGetLiveness(t *testing.T) {
-	resultStr := `{
-   "jsonrpc":"2.0",
-   "result":{
-      "checks":{
-         "network.validators.heartbeat":{
-            "message":{
-               "heartbeat":1591041377
+	tests := []struct {
```

Comment:
There's a lot going on here - we're inline declaring a struct, then declaring an array of that struct, and then inline populating that array. To make this more readable, can we break the struct definition out into a separate package-private named type?

Reply:
Updating